install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -314,7 +314,10 @@ data:
 
 {{- if .Values.global.kubeProxyReplacement }}
   kube-proxy-replacement:  {{ .Values.global.kubeProxyReplacement | quote }}
-{{- end}}
+{{- end }}
+{{- if .Values.global.enableHealthCheckNodeport }}
```

Comment:
Could this be grouped with other `global.nodePort` flags? E.g. `global.nodePort.enableHealthCheck`?

Reply:
I still prefer to keep it outside, since in kube-proxy 'partial'  mode the nodeport option is not enabled. What do you think?

Reply:
The healthcheck nodeport is only relevant for the bpf nodeport, therefore I suggested to group them together.

Reply:
@brb I have one question here, I think we need to pass in the value of 'enableHealthCheckNodeport'  irrespective of the Nodeport is enabled from the helm chart or not.